### PR TITLE
Fixed UT

### PR DESCRIPTION
**Pull request by chenzhitong**
close https://github.com/neo-project/neo-modules/issues/419

### CODE REVIEW

**Comment by chenzhitong** [comment]
Draft

File: src/RpcClient/ContractClient.cs [hunk 1]
```diff
@@ -57,7 +57,7 @@ public async Task<Transaction> CreateDeployContractTxAsync(byte[] contractScript
             byte[] script;
             using (ScriptBuilder sb = new ScriptBuilder())
             {
-                sb.EmitSysCall(ApplicationEngine.System_Contract_Create, contractScript, manifest.ToString());
+                sb.EmitSysCall(ApplicationEngine.System_Contract_CreateStandardAccount, contractScript, manifest.ToString());
```
Comment (shargon) [inline src/RpcClient/ContractClient.cs:60]: Rename contractScript to nefFile?

Comment (shargon) [reply src/RpcClient/ContractClient.cs:60]: It's not `System.Contract.CreateStandardAccount`

Comment (chenzhitong) [reply src/RpcClient/ContractClient.cs]: Fixed
```c#
sb.EmitSysCall(ApplicationEngine.System_Contract_Call, NativeContract.Management.Hash, "deploy", new byte[1], manifest.ToString());
```

**Review by shargon — DISMISSED** [review]

**Comment by superboyiii** [comment]
I think this is the same as https://github.com/neo-project/neo-modules/pull/422.

**Review by shargon — APPROVED** [review]

**Pull request merged**